core/node/core.go

```diff
@@ -97,11 +103,61 @@ func OnlineExchange(provide bool) interface{} {
 				return exch.Close()
 			},
 		})
+		lc.Append(fx.Hook{
+			OnStart: func(ctx context.Context) error {
+
+				go func() {
+					for cbs != nil {
```

Comment:
@schomatis I changed this to use the config, this works so long as the tester has Internal.Bitswap setup in the config, and does nothing otherwise.

Reply:
@coryschwartz Great, thanks! Just to confirm if I understand this correctly, a change through the `ipfs config` CLI to these configurations will be reflected on the fly in the `*config.InternalBitswap` pointer here without needing to restart the daemon, right?